sdk/keyvault/Azure.Security.KeyVault.Certificates/src/KeyVaultCertificateWithPolicy.cs

```diff
@@ -21,6 +22,12 @@ internal KeyVaultCertificateWithPolicy(CertificateProperties properties = null)
         /// </summary>
         public CertificatePolicy Policy { get; internal set; }
 
+        /// <summary>
+        /// Gets a value indicating whether the certificate chain preserves its original order.
+        /// The default value is false, which sets the leaf certificate at index 0.
+        /// </summary>
+        public bool? PreserveCertOrder { get; internal set; }
```

Comment:
```suggestion
        public bool? PreserveCertificateOrder { get; internal set; }
```

Please see previous comments for context.